Rezept.md

```diff
@@ -1,6 +1,6 @@
 # Gugelhupf
 
-Ein Rezept für einen Gugelhupf.
+Ein Rezept für einen [Gugelhupf](https://de.wikipedia.org/wiki/Gugelhupf).
```

Comment:
Please link english version of wiki site since german speaking people most likely know what a Gugelhupf is

Reply:
ok, I'll link the English wikipedia in your German text  ☕️

Reply:
English speaking people have as much rights to Gugelhupf as german speaking ones